### DISCUSSION TITLE
Grouping targets drop down by FOLDER property

### DISCUSSION THREAD

**Discussion by itzandroidtab** [microsoft/vscode-cmake-tools#3949] [Ideas]
I am currently working on a project that has over 15 targets. I was thinking it would be a great idea to support grouping the targets based on the folder property in cmake. I was thinking of something like this. When you click on the folder you would see all the targets in that specific folder:
![image](https://github.com/user-attachments/assets/3eb4ec2a-e43c-45df-92be-c2851e0d6c40)

Currently it is difficult to find the target I want to build:
![image](https://github.com/user-attachments/assets/2d128bf7-d024-4222-a241-86cc1ae11912)

Information about the folder property
https://cmake.org/cmake/help/latest/prop_tgt/FOLDER.html

**Comment by gcampbell-msft** [comment]
I'll close this in favor of tracking your PR here: #3953 @itzandroidtab
